### DISCUSSION TITLE
Animal sewer shouldn't need energy

### DISCUSSION THREAD

**Discussion by kene02** [InnovativeOnlineIndustries/Industrial-Foregoing#1114] [Completed/Done]
To Whom It May Concern,

Can someone please make it so that the animal sewer doesn't need energy to operate? The one in Minefactory Reloaded doesn't need any and having to give it both an energy connection (a fluxduct) AND a fluid connection (a fluiduct) is a logistical nightmare if you consider how it is placed on the ground with all four of its sides in the horizontal direction covered in dirt.

Thanks.

Sincerely yours,
Ken

**Comment by Buuz135** [comment]
Its able to have a range, with that it needs power for balance
